float  confidence 
uint32 robot_id      
float  x         
float  y           
float  orientation        
float  pixel_x    
float  pixel_y  
float  height   
